datarepo-clienttests/src/main/java/scripts/testscripts/DRSLookup.java

```diff
@@ -43,6 +44,13 @@ public DRSLookup() {
   private static List<BlobId> scratchFiles = new ArrayList<>();
   private String dirObjectId;
 
+  private int K = 1;
```

Comment:
should this be an `Integer` since technically it could be null?  Also, probably want a better name